cpp/src/parquet/file_reader.h

```diff
@@ -44,7 +44,8 @@ class PARQUET_EXPORT RowGroupReader {
   // An implementation of the Contents class is defined in the .cc file
   struct Contents {
     virtual ~Contents() {}
-    virtual std::unique_ptr<PageReader> GetColumnPageReader(int i) = 0;
+    virtual std::unique_ptr<PageReader> GetColumnPageReader(
```

Comment:
`GetColumnChunkRange` is just an internal implementation to roughly get the read range of column chunk. In fact, the exact range of a column chunk can be more complex if IndexPage and ColumnMetaData are considered. It would be a burden to make it public.